src/memgraph.cpp

```diff
@@ -598,6 +599,43 @@ int main(int argc, char **argv) {
   memgraph::query::procedure::gModuleRegistry.UnloadAndLoadModulesFromDirectories();
   memgraph::query::procedure::gCallableAliasMapper.LoadMapping(FLAGS_query_callable_mappings_path);
 
+  std::vector<memgraph::storage::VectorIndexSpec> vector_index_specs;
+  if (!FLAGS_experimental_vector_indexes.empty()) {
+    const auto specs = memgraph::utils::Split(FLAGS_experimental_vector_indexes, ";");
+    if (!specs.empty()) {
+      auto storage = db_acc->Access();
+      vector_index_specs.reserve(specs.size());
+      for (const auto &spec : specs) {
+        const auto an_index_split = memgraph::utils::Split(spec, "__");
+        if (an_index_split.size() != 3 && an_index_split.size() != 4) {
+          LOG_FATAL(
+              "--experimental-vector-indexes is not in the right format to use vector indexes. Use "
+              "Label__property__\{JSON\},... format instead.");
+        }
+        const auto &label_name = an_index_split[1];
+        const auto &property_name = an_index_split[2];
+        const auto label_id = storage->NameToLabel(label_name);
+        const auto property_id = storage->NameToProperty(property_name);
+        if (an_index_split.size() == 3) {
+          vector_index_specs.emplace_back(memgraph::storage::VectorIndexSpec{
+              .index_name = an_index_split[0], .label = label_id, .property = property_id, .config = {}});
+        }
+        if (an_index_split.size() == 4) {
+          vector_index_specs.emplace_back(
+              memgraph::storage::VectorIndexSpec{.index_name = an_index_split[0],
+                                                 .label = label_id,
+                                                 .property = property_id,
+                                                 .config = nlohmann::json::parse(an_index_split[3])});
+        }
+      }
```

Comment:
If nothing is provided as a config, let's provide better error messages or provide all default values

![Screenshot 2024-10-23 at 8 44 07 AM](https://github.com/user-attachments/assets/97d72a57-5ecb-4aa3-853f-26cb77f5126e)

![Screenshot 2024-10-23 at 8 46 16 AM](https://github.com/user-attachments/assets/9ae52c9b-6457-4182-979c-fb5c82017f0b)